include/openmc/particle.h

```diff
@@ -61,6 +61,16 @@ class Particle : public ParticleData {
   void event_revive_from_secondary();
   void event_death();
 
+  //! needed for determining the value of the pulse-height tally
+  //
+  //! sums up the energy a photon looses after in a cell
+  void energy_delivert_in_cell();
```

Comment:
looses -> loses

And when you put "delivert" did you mean delivered?

Reply:
Yes in any case, I have changed directly.